synapse/lib/storm.py

```diff
@@ -278,6 +282,16 @@
                 }
             ]
         },
+        'cmdinput': {
```

Comment:
Do we want to also include the output forms when using `--yield` or should that be covered in the docs?

Reply:
I haven't decided yet :)  Right now, since a `--yield` case typically causes only one type of node to be yielded, it's been easy enough to just have the command help show that.